pkg/kubectl/cmd/exec.go

```diff
@@ -135,19 +139,19 @@ type ExecOptions struct {
 func (p *ExecOptions) Complete(f *cmdutil.Factory, cmd *cobra.Command, argsIn []string, argsLenAtDash int) error {
 	// Let kubectl exec follow rules for `--`, see #13004 issue
 	if len(p.PodName) == 0 && (len(argsIn) == 0 || argsLenAtDash == 0) {
-		return cmdutil.UsageError(cmd, "POD is required for exec")
+		return cmdutil.UsageError(cmd, execUsageStr)
 	}
 	if len(p.PodName) != 0 {
-		printDeprecationWarning("exec POD", "-p POD")
+		printDeprecationWarning("exec <pod-name>", "-p <pod-name>")
```

Comment:
Keep using uppercase for arguments for consistency: `printDeprecationWarning("exec POD", "-p POD")`. Also `<...>` denotes an optional argument, which is not the case here.